src/embeddings/metadata_prompt_builder.py

```
@@ -0,0 +1,162 @@
+"""
+Metadata-Based Prompt Builder for Jina v4
+
+Simple, clean approach: take metadata and build human-readable prompts
+that clearly state our intent and provide context.
+"""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)